frame/society/src/lib.rs

```diff
@@ -869,15 +860,12 @@ pub mod pallet {
 		/// Parameters:
 		/// - `pos`: Position in the `Bids` vector of the bid who should be unvouched.
 		///
-		/// # <weight>
+		/// Total Complexity: O(B)
 		/// Key: B (len of bids)
 		/// - One storage read O(1) to check the signer is a vouching member.
 		/// - One storage mutate to retrieve and update the bids. O(B)
```

Comment:
Looks like this still needs to be removed? Same for the other extrinsics in this file.